docs/repo-guides/release-nnf-sw/readme.md

```diff
@@ -155,14 +155,14 @@ that everything is current on `master` for `nnf-deploy`.
     git submodule foreach 'git checkout `git describe --match="v*" HEAD`'
     ```
 
-6. Verify that each submodule is now at the proper tagged version.
+6. Add all submodules with `git add` for each submodule.
```

Comment:
If you don't add the submodules to the commit, `git submodule` command doesn't show what you are looking for in step 7.